docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -85,28 +87,20 @@ spec:
     metrics:
       elasticsearchRefs:
       - name: monitoring
-        namespace: observability <1>
+        namespace: observability <2>
     logs:
       elasticsearchRefs:
       - name: monitoring
-        namespace: observability <1>
+        namespace: observability <2>
 ----
 
-<1> The use of `namespace` is optional if the monitoring Elasticsearch cluster and the monitored Elastic Stack resource are running in the same namespace.
-
-NOTE: You can configure an Elasticsearch cluster to monitor itself.
+<1> The same monitoring cluster is used for metrics and logs, but seperate clusters could be used.
 
-NOTE: If Stack Monitoring is configured for a Beat, but the corresponding Elasticsearch cluster is not monitored, the Kibana Stack Monitoring page will not show the Beats data.
+<2> The use of `namespace` is optional if the monitoring Elasticsearch cluster and the monitored Elastic Stack resource are running in the same namespace.
 
 NOTE: If Logs Stack Monitoring is configured for a Beat, and custom container arguments (`podTemplate.spec.containers[].args`) include `-e`, which enables logging to stderr and disables log file output, this argument will be removed from the Pod to allow the Filebeat sidecar to consume the Beat's log files.
 
-NOTE: Stack Monitoring for Logstash on ECK is only available for Logstash versions 8.7.0 and later.
-
-IMPORTANT: The monitoring cluster must be managed by ECK in the same Kubernetes cluster as the monitored one.
-
-You can send metrics and logs to two different Elasticsearch monitoring clusters.
-
-You can also enable Stack Monitoring on Elasticsearch only or on Kibana only. In the latter case, Kibana will not be available on the Stack Monitoring Kibana page (check link:https://www.elastic.co/guide/en/kibana/current/monitoring-data.html#monitoring-data[View monitoring data in Kibana]).
+You can also enable Stack Monitoring on a single stack component only. In case Elasticsearch is not monitored, other Stack components will not be available on the Stack Monitoring Kibana page (check link:https://www.elastic.co/guide/en/kibana/current/monitoring-data.html#monitoring-data[View monitoring data in Kibana]).
```

Comment:
```suggestion
You can also enable Stack Monitoring on a single Stack component only. In case Elasticsearch is not monitored, other Stack components will not be available on the Stack Monitoring Kibana page (check link:https://www.elastic.co/guide/en/kibana/current/monitoring-data.html#monitoring-data[View monitoring data in Kibana]).
```